servers/partner-ci/builders/vcenter.test.sh

```diff
@@ -62,7 +62,7 @@ if [[ $ISO_FILE == *"Mirantis"* ]]; then
   export FUEL_RELEASE=$(echo $ISO_FILE | cut -d- -f2 | tr -d '.iso')
 fi
 
-export ENV_NAME="${ENV_PREFIX}.${SNAPSHOTS_ID:?}"
```

Comment:
`line 65: SNAPSHOTS_ID: parameter null or not set ` 
`export SNAPSHOTS_ID= ` - it's the problem
```
+ [[ 610 == *\l\a\s\t\S\u\c\c\e\s\s\f\u\l\B\u\i\l\d* ]]
+ export SNAPSHOTS_ID= << it's the problem
+ SNAPSHOTS_ID=
+ '[' -f build.plugin_version ']'
++ grep PLUGIN_VERSION
++ cut -d= -f2
+ version=3.2.0
+ export DVS_PLUGIN_VERSION=3.2.0
+ DVS_PLUGIN_VERSION=3.2.0
+ '[' -z 116 ']'
+ '[' -f nosetest.xml ']'
+ sudo rm -rf 'logs/*'
+ [[ MirantisOpenStack-9.0.iso == *\M\i\r\a\n\t\i\s* ]]
++ echo MirantisOpenStack-9.0.iso
++ tr -d .iso
++ cut -d- -f2
+ export FUEL_RELEASE=90
+ FUEL_RELEASE=90
/tmp/hudson722437583800698906.sh: line 65: SNAPSHOTS_ID: parameter null or not set
```